README.md

```diff
@@ -2,12 +2,12 @@
 
 ## Description
 
-This target audience for this course is the Scala Application Developer Community. So, there's a focus on people who use the language to develop 'end-user' applications and not necessarily those developers who design and write libraries, toolkits or frameworks (although they may benefit from what's in this course if they haven't already worked with Dotty).
+The target audience for this course is the Scala _Application_ Developer Community. In other words, the focus is on people who use the language to develop 'end-user' applications and not necessarily those developers who design and write libraries, toolkits or frameworks (although they may benefit from what is in this course if they haven't already worked with Dotty).
 
-The goal of the course is to explore the new features in the Scala language brought by [the Dotty project](https://dotty.epfl.ch). Even though it's still early days and the first release candidate of the Scala 3 language is probably out for another 6 months, it is our believe that one can never start too early by learning Dotty for the following reasons:
+The goal of the course is to explore the new features in the Scala language brought by [the Dotty project](https://dotty.epfl.ch). Even though it is still early days and the first release candidate of the Scala 3 language is probably not out for another 6 months, it is our believe that one can never start too early to learn Dotty for the following reasons:
 
-- There's a lot to learn, so if you wait until everything is ready, you're already too late.
-- There are many opportunities to contribute to the Dotty project, which will accelerate it's development, exercise it which will inevitably expose issues that can be corrected well in time. Using Dotty will provide opportunities to do this.
+- There is a lot to learn, so if you wait until everything is ready, you risk having to play catchup.
+- There are many opportunities to contribute to the Dotty project, which will accelerate its development in the coming months. Using Dotty will provide opportunities to identity issues to be fixed sooner rather than later.
```

Comment:
`Using Dotty will provide opportunities to identity issues`
 seems a typo here, Is this supposed to be **identify** instead of **identity**